ts/src/renderer/phaser/classes/PhaserAnimatedEntity.ts

```diff
@@ -12,17 +12,53 @@ class PhaserAnimatedEntity extends PhaserEntity {
 		const bounds = entity._bounds2d;
 		const sprite = this.sprite = scene.add.sprite(0, 0, key);
 		sprite.setDisplaySize(bounds.x, bounds.y);
+
+		// {0,1} is 0 radians | {0,-1} is pi radians
```

Comment:
yup, that is how radians work. can we remove it?

Reply:
`false`